src/getMovedFile.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function getMovedFile(source, destination) {
+  if (!fs.existsSync(destination)) {
+    console.error(`Destination directory ${destination} does not exist.`);
+    process.exit(1);
+  }
+
+  const sourceStats = fs.statSync(source);
```

Comment:
I'm not sure why but here I'm getting an error, `ENOENT: no such file or directory, stat 'file.txt'`  it works properly only with `path.join(__dirname, source)`

Reply:
Thank you for the comment, I really didn't understand this topic very well, had to spend some time reading various materials, but finally everything is working correctly.